test/UnitTest/System.Web.OData.Test/PublicApi/System.Web.OData.PublicApi.bsl

```diff
@@ -3216,6 +3216,9 @@ public class System.Web.OData.Formatter.Serialization.SelectExpandNode {
 	public static void GetStructuralProperties (Microsoft.OData.Edm.IEdmStructuredType structuredType, System.Collections.Generic.HashSet`1[[Microsoft.OData.Edm.IEdmStructuralProperty]] structuralProperties, System.Collections.Generic.HashSet`1[[Microsoft.OData.Edm.IEdmStructuralProperty]] nestedStructuralProperties)
 }
 
+[
+JsonConverterAttribute(),
```

Comment:
This seems to be the only instance of this attribute in our public API.

Reply:
In #1078 PeopleController.cs (https://github.com/jvitor83/OData.Sample/blob/master/OData.Sample/Controllers/PeopleController.cs) Is derived from ApiController instead of ODataController as  a result OData serialization will not be used. Instead default ASP.NET json serialization will happen and it uses newtonsoft.json.

Please, take a look on https://github.com/OData/WebApi/blob/master/src/System.Web.OData/OData/Query/Expressions/SelectExpandWrapper.cs it uses the same approach

Reply:
SelectExpandWrapper that uses converter too is internal => not visible for Public API.
I can make DynamicTypeWrapper internal too, but it will breaking change for public API

Reply:
I moved to class that doing real job. Public API contains not changes now